src/com/rockbite/bootcamp/Type.java

```diff
@@ -0,0 +1,28 @@
+package com.rockbite.bootcamp;
+
+import java.io.Serializable;
+
+/**
+ * Type of Items
+ * it also implements Serializable like Item class, because it is part of Item class
+ */
+public class Type implements Serializable {
```

Comment:
Type is very generic, and is already used in many core libraries including the JVM itself.

ItemType would be a better name in this case